pypika/__init__.py

```diff
@@ -100,7 +100,7 @@
 
 __author__ = "Timothy Heys"
 __email__ = "theys@kayak.com"
-__version__ = "0.48.9"
+__version__ = "0.48.10"
```

Comment:
is this the expected version update?

Reply:
I suggest to bump the version separately and also start a `CHANGELOG.md` as well as describe all of the changes there

Reply:
I reverted the version back to 0.48.9 So no version change. 

Have you guys considered using the automated release notes? https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes 
It'd be based on the PR labels. Might be easy way to automate 😄